### PR TITLE
Support Android Studio 3.0

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 8 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AndroidUtils.java#L89](https://github.com/SonarSource/sonar-scanner-gradle/blob/456a2bf83b68207968af893af0f253e0a6b9245a/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L89): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L115](https://github.com/SonarSource/sonar-scanner-gradle/blob/456a2bf83b68207968af893af0f253e0a6b9245a/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L115): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L134](https://github.com/SonarSource/sonar-scanner-gradle/blob/456a2bf83b68207968af893af0f253e0a6b9245a/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L134): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L135](https://github.com/SonarSource/sonar-scanner-gradle/blob/456a2bf83b68207968af893af0f253e0a6b9245a/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L135): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L161](https://github.com/SonarSource/sonar-scanner-gradle/blob/456a2bf83b68207968af893af0f253e0a6b9245a/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L161): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/456a2bf83b68207968af893af0f253e0a6b9245a/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/456a2bf83b68207968af893af0f253e0a6b9245a/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L226](https://github.com/SonarSource/sonar-scanner-gradle/blob/456a2bf83b68207968af893af0f253e0a6b9245a/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L226): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Shusshu** [comment]
@henryju I think you missed the PR waiting for 2 weeks already.... https://github.com/SonarSource/sonar-scanner-gradle/pull/33

**Comment by SonarTech** [comment]
SonarQube analysis reported 8 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AndroidUtils.java#L89](https://github.com/SonarSource/sonar-scanner-gradle/blob/60492927d34a92e5c2e90eef7fc89bdb4bc2b358/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L89): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L115](https://github.com/SonarSource/sonar-scanner-gradle/blob/60492927d34a92e5c2e90eef7fc89bdb4bc2b358/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L115): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L134](https://github.com/SonarSource/sonar-scanner-gradle/blob/60492927d34a92e5c2e90eef7fc89bdb4bc2b358/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L134): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L135](https://github.com/SonarSource/sonar-scanner-gradle/blob/60492927d34a92e5c2e90eef7fc89bdb4bc2b358/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L135): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L161](https://github.com/SonarSource/sonar-scanner-gradle/blob/60492927d34a92e5c2e90eef7fc89bdb4bc2b358/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L161): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/60492927d34a92e5c2e90eef7fc89bdb4bc2b358/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/60492927d34a92e5c2e90eef7fc89bdb4bc2b358/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L226](https://github.com/SonarSource/sonar-scanner-gradle/blob/60492927d34a92e5c2e90eef7fc89bdb4bc2b358/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L226): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 8 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AndroidUtils.java#L89](https://github.com/SonarSource/sonar-scanner-gradle/blob/1781312ebd0033e054573782e0f317106039d3d4/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L89): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L115](https://github.com/SonarSource/sonar-scanner-gradle/blob/1781312ebd0033e054573782e0f317106039d3d4/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L115): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L134](https://github.com/SonarSource/sonar-scanner-gradle/blob/1781312ebd0033e054573782e0f317106039d3d4/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L134): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L135](https://github.com/SonarSource/sonar-scanner-gradle/blob/1781312ebd0033e054573782e0f317106039d3d4/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L135): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L161](https://github.com/SonarSource/sonar-scanner-gradle/blob/1781312ebd0033e054573782e0f317106039d3d4/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L161): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/1781312ebd0033e054573782e0f317106039d3d4/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/1781312ebd0033e054573782e0f317106039d3d4/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L226](https://github.com/SonarSource/sonar-scanner-gradle/blob/1781312ebd0033e054573782e0f317106039d3d4/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L226): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/org/sonarqube/gradle/AndroidUtils.java [hunk 1]
```diff
@@ -181,7 +181,7 @@ static BaseVariant findVariant(List<BaseVariant> candidates, @Nullable String te
       if (result.isPresent()) {
         return result.get();
       } else {
-        throw new IllegalArgumentException("Unable to find variant '" + userConfiguredBuildVariantName + "' to use for SonarQube configuration");
+        throw new IllegalArgumentException("Unable to find variant '" + userConfiguredBuildVariantName + "' to use for SonarQube analysis. Candidates are: " + candidates.stream().map(BaseVariant::getName).collect(Collectors.joining(", ")));
```
Comment (SonarTech) [inline src/main/java/org/sonarqube/gradle/AndroidUtils.java]: MINOR Split this 240 characters long line (which is greater than 180 authorized). [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)

**Comment by SonarTech** [comment]
SonarQube analysis reported 9 issues
* ![MINOR][MINOR] 9 minor

Watch the comments in this conversation to review them.

#### 8 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AndroidUtils.java#L89](https://github.com/SonarSource/sonar-scanner-gradle/blob/ccdad0a40e6e5db73dd9fa3955be3af3a6e72d08/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L89): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L115](https://github.com/SonarSource/sonar-scanner-gradle/blob/ccdad0a40e6e5db73dd9fa3955be3af3a6e72d08/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L115): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L134](https://github.com/SonarSource/sonar-scanner-gradle/blob/ccdad0a40e6e5db73dd9fa3955be3af3a6e72d08/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L134): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L135](https://github.com/SonarSource/sonar-scanner-gradle/blob/ccdad0a40e6e5db73dd9fa3955be3af3a6e72d08/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L135): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L161](https://github.com/SonarSource/sonar-scanner-gradle/blob/ccdad0a40e6e5db73dd9fa3955be3af3a6e72d08/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L161): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/ccdad0a40e6e5db73dd9fa3955be3af3a6e72d08/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/ccdad0a40e6e5db73dd9fa3955be3af3a6e72d08/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L226](https://github.com/SonarSource/sonar-scanner-gradle/blob/ccdad0a40e6e5db73dd9fa3955be3af3a6e72d08/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L226): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 9 issues
* ![MINOR][MINOR] 9 minor

Watch the comments in this conversation to review them.

#### 8 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AndroidUtils.java#L89](https://github.com/SonarSource/sonar-scanner-gradle/blob/ee0034d72f43c3a25feb42744a366384d7261f18/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L89): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L115](https://github.com/SonarSource/sonar-scanner-gradle/blob/ee0034d72f43c3a25feb42744a366384d7261f18/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L115): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L134](https://github.com/SonarSource/sonar-scanner-gradle/blob/ee0034d72f43c3a25feb42744a366384d7261f18/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L134): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L135](https://github.com/SonarSource/sonar-scanner-gradle/blob/ee0034d72f43c3a25feb42744a366384d7261f18/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L135): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L161](https://github.com/SonarSource/sonar-scanner-gradle/blob/ee0034d72f43c3a25feb42744a366384d7261f18/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L161): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/ee0034d72f43c3a25feb42744a366384d7261f18/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/ee0034d72f43c3a25feb42744a366384d7261f18/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L226](https://github.com/SonarSource/sonar-scanner-gradle/blob/ee0034d72f43c3a25feb42744a366384d7261f18/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L226): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 9 issues
* ![MINOR][MINOR] 9 minor

Watch the comments in this conversation to review them.

#### 8 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AndroidUtils.java#L89](https://github.com/SonarSource/sonar-scanner-gradle/blob/34901425ad2261cf0ce8a834d49da5ad1138ef0d/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L89): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L115](https://github.com/SonarSource/sonar-scanner-gradle/blob/34901425ad2261cf0ce8a834d49da5ad1138ef0d/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L115): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L134](https://github.com/SonarSource/sonar-scanner-gradle/blob/34901425ad2261cf0ce8a834d49da5ad1138ef0d/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L134): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L135](https://github.com/SonarSource/sonar-scanner-gradle/blob/34901425ad2261cf0ce8a834d49da5ad1138ef0d/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L135): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L161](https://github.com/SonarSource/sonar-scanner-gradle/blob/34901425ad2261cf0ce8a834d49da5ad1138ef0d/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L161): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/34901425ad2261cf0ce8a834d49da5ad1138ef0d/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/34901425ad2261cf0ce8a834d49da5ad1138ef0d/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L226](https://github.com/SonarSource/sonar-scanner-gradle/blob/34901425ad2261cf0ce8a834d49da5ad1138ef0d/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L226): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 8 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AndroidUtils.java#L89](https://github.com/SonarSource/sonar-scanner-gradle/blob/1c99bea77f827b3e881aa1d603c9d491d4f8e924/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L89): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L115](https://github.com/SonarSource/sonar-scanner-gradle/blob/1c99bea77f827b3e881aa1d603c9d491d4f8e924/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L115): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L134](https://github.com/SonarSource/sonar-scanner-gradle/blob/1c99bea77f827b3e881aa1d603c9d491d4f8e924/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L134): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L135](https://github.com/SonarSource/sonar-scanner-gradle/blob/1c99bea77f827b3e881aa1d603c9d491d4f8e924/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L135): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L161](https://github.com/SonarSource/sonar-scanner-gradle/blob/1c99bea77f827b3e881aa1d603c9d491d4f8e924/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L161): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/1c99bea77f827b3e881aa1d603c9d491d4f8e924/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L162](https://github.com/SonarSource/sonar-scanner-gradle/blob/1c99bea77f827b3e881aa1d603c9d491d4f8e924/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L162): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [AndroidUtils.java#L227](https://github.com/SonarSource/sonar-scanner-gradle/blob/1c99bea77f827b3e881aa1d603c9d491d4f8e924/src/main/java/org/sonarqube/gradle/AndroidUtils.java#L227): Remove this use of "Nullable"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**